src/server/config.ts

```diff
@@ -0,0 +1,2 @@
+export const GITHUB_CALLBACK_URL = '/api/auth/github/callback';
+export const GOOGLE_CALLBACK_URL = 'http://localhost:8081/api/auth/google/callback';
```

Comment:
Is this a placeholder until we get Google working?

Reply:
Ahh. Mistake. Done.